x-pack/plugins/aiops/public/components/explain_log_rate_spikes/explain_log_rate_spikes_analysis.tsx

```diff
@@ -82,6 +99,7 @@ export const ExplainLogRateSpikesAnalysis: FC<ExplainLogRateSpikesAnalysisProps>
     if (onSelectedChangePoint) {
       onSelectedChangePoint(null);
     }
+    setShouldRerunAnalysis(false);
```

Comment:
This should be replaced with `setCurrentAnalysisWindowParameters(windowParameters);` similar to the call in the `useEffect` that trigger the analysis on load.

Reply:
Good catch - updated in 3f2a5c5034db76c0767c0c7cf5ee570214be066c

Reply:
It looks like that fix wasn't commited/pushed properly, I cannot see it being changed in the latest version, can you try again?